design.md

```diff
@@ -2,12 +2,24 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+Lord of the Rings fans who are moderately well-versed in the lore of the world.
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+Because it is fun (we are LotR fans) and because it also provides essentially equivalent functionality without major changes to the program model.
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+None.  We essentially took the existing syntax and made it more verbose.  Yay!
+
```

Comment:
While what we did would probably make no prospective user (except maybe a diehard LOTR fan) happy, I wouldn't say that making the language more verbose is always bad.

Take a look at this piconot implementation from last year: https://github.com/hmc-cs111-fall2014/piconot/pull/10/files. I think that making the syntax more verbose made the language easier to understand, which is a good thing for some use cases.